### DISCUSSION TITLE
disable cardplayer when not playing anything.

### DISCUSSION THREAD

**Discussion by iorifly** [frenck/spook#803] [Q&A]
Hello, how are you?

Guys, I installed spook via HACS and now I'm trying to disable a media card, in case nothing is being played. Could someone help me?

`
type: media-control
entity: media_player.echo_show_8_2a_geracao_de_renato
card_mod:
  style: |
    :host {
      {% if is_state('media_player.echo_show_8_2a_geracao_de_renato','playing') %}
        homeassistant.enable_entity.media_player.echo_show_8_2a_geracao_de_renato
        
        
        {% elseif %}
        
        homeassistant.disable_entity.media_player.echo_show_8_2a_geracao_de_renato
         
     
        
      {% endif %}
    }
`

**Comment by iorifly** [comment]
I also tried this option without success

> 
type: media-control
entity: media_player.echo_show_8_2a_geracao_de_renato
card_mod:
  style: |
    :host {
      {% if is_state('media_player.echo_show_8_2a_geracao_de_renato','playing') %}
          service: homeassistant.unhide_entity
          data:
            entity_id: media_player.echo_show_8_2a_geracao_de_renato
              
        {% elif is_state('media_player.echo_show_8_2a_geracao_de_renato','standby') %}

          service: homeassistant.hide_entity
          data:
            entity_id: media_player.echo_show_8_2a_geracao_de_renato
        
      {% endif %}
    }
>

**Comment by frenck** [comment]
> disable a media card

You cannot disable cards with Spook.

../Frenck
